proxyclient/m1n1/adt.py

```diff
@@ -228,7 +228,6 @@
     "unk3" / Hex(Int8ul),
     "r0h" / Hex(Int8ul),
     "r0l" / Hex(Int8ul),
-    "unk4" / Hex(Int8ul),
```

Comment:
This depends unfortunately on the iboot version. Depending on the version we need variants with either 3 or 4 bytes of additional unknown data (maybe padding).

Reply:
Hmm, so what's the proper fix here, then?  Figure out what iBoot version changed it, and check that in the parser?

Reply:
I would add this as additional variant and decide based on the length which one to use. The length of the property has to be a multiple of the element length and based on the size and typical number of elements I think we're not going to run into ambiguous situations.

Reply:
Done.